doc: release: know_issues: Order of update matters for nrf5340 (#30274)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

  **Affected platforms:** nRF5340

.. rst-class:: v3-4-0 v3-3-2 v3-3-1 v3-3-0 v3-2-5 v3-2-4 v3-2-3 v3-2-2 v3-2-1 v3-2-0 v3-1-1 v3-1-0 v3-0-2 v3-0-1 v3-0-0 v2-9-3 v2-9-2 v2-9-1 v2-9-0 v2-8-0 v2-7-0 v2-6-6 v2-6-5 v2-6-4 v2-6-3 v2-6-2 v2-6-1 v2-6-0

NCSDK-34106: DFU malfunction if app is activated before NETCORE

>   While marking Firmware update with MCUmgr maintain order of applying boot flags: APPCORE, NETCORE.

Reviewer: ```suggestion
NCSDK-34106: DFU malfunction if the application core is activated before network core
  When performing a firmware update using MCUmgr, the order in which the boot flags are applied must be maintained.
  Activating the application core image before the network core image causes MCUboot to skip the network core update.
```
This is my interpretation of the original text. Please check if that's technically correct.

Also, isn't there a workaround we can document here?

Author: The workaround is to maintain the order.

Reviewer: Let me push another suggestion then

>   Activating the application core image before the network core image causes MCUboot to skip the network core update.

Reviewer: ```suggestion
  When performing a firmware update using MCUmgr, if you activate the application core image before the network core image, MCUboot skips the network core update.
```

  **Affected platforms:** nRF5340

> 

Reviewer: ```suggestion

**Workaround:** Maintain the correct order in which the boot flags are applied
```

> **Workaround:** Maintain the correct order in which the boot flags are applied

Reviewer 2: ```suggestion
  **Workaround:** Maintain the correct order in which the boot flags are applied.
```

Reviewer 2: If at all possible, could you add the wontfix label in the issue NCSDK-28152?

  **Affected platforms:** nRF5340

> .. rst-class:: wantfix v3-4-0 v3-3-2 v3-3-1 v3-3-0 v3-2-5 v3-2-4 v3-2-3 v3-2-2 v3-2-1 v3-2-0 v3-1-1 v3-1-0 v3-0-2 v3-0-1 v3-0-0 v2-9-3 v2-9-2 v2-9-1 v2-9-0 v2-8-0 v2-7-0 v2-6-6 v2-6-5 v2-6-4 v2-6-3 v2-6-2 v2-6-1 v2-6-0

Reviewer 2: ```suggestion
.. rst-class:: wontfix v3-4-0 v3-3-2 v3-3-1 v3-3-0 v3-2-5 v3-2-4 v3-2-3 v3-2-2 v3-2-1 v3-2-0 v3-1-1 v3-1-0 v3-0-2 v3-0-1 v3-0-0 v2-9-3 v2-9-2 v2-9-1 v2-9-0 v2-8-0 v2-7-0 v2-6-6 v2-6-5 v2-6-4 v2-6-3 v2-6-2 v2-6-1 v2-6-0
```

Reviewer 2: Is this intentional? Asking you to add wontfix in the issue NCSDK-28152.

Author: > Is this intentional? Asking you to add wontfix in the issue NCSDK-28152.

No. I do my mistakes by mistake. Heh, but must admit that this was quite Freudian. The 28152 is not mine, I do not know what that is.

  **Affected platforms:** nRF5340

.. rst-class:: wontfix v3-4-0 v3-3-2 v3-3-1 v3-3-0 v3-2-5 v3-2-4 v3-2-3 v3-2-2 v3-2-1 v3-2-0 v3-1-1 v3-1-0 v3-0-2 v3-0-1 v3-0-0 v2-9-3 v2-9-2 v2-9-1 v2-9-0 v2-8-0 v2-7-0 v2-6-6 v2-6-5 v2-6-4 v2-6-3 v2-6-2 v2-6-1 v2-6-0

NCSDK-34106: DFU malfunction if the application core is activated before network core

>   When performing a firmware update using MCUmgr, if you activate the application core image before the network core image, MCUboot skips the network core update.

Reviewer 2: ```suggestion
  If you activate the application core image before the network core image when performing a firmware update using MCUmgr, MCUboot skips the network core update.
```
Still asking you to add the wontfix label to NCSDK-28152 even though it is not your cup of tea. This has been agreed with the actual owner of the issue offline.

Author: Added.
